docs: add krew plugin to the troubleshooting docs (#10768)
rook/rook: .github/ISSUE_TEMPLATE/bug_report.md

**File(s) to submit**:

* Cluster CR (custom resource), typically called `cluster.yaml`, if necessary

**Logs to submit**:

* Operator's logs, if necessary
* Crashing pod(s) logs, if necessary

  To get logs, use `kubectl -n <namespace> logs <pod name>`
  When pasting logs, always surround them with backticks or use the `insert code` button from the Github UI.
  Read [GitHub documentation if you need help](https://help.github.com/en/articles/creating-and-highlighting-code-blocks).

**Cluster Status to submit**:

* Output of krew commands, if necessary

>   To get the health of the cluster, use `kubectl rook_ceph health`

Reviewer: ```suggestion
  To get the health of the cluster, use `kubectl rook-ceph health`
```

  To get the status of the cluster, use `kubectl rook-ceph ceph status`

>   For other commands and how to use Krew Read [Krew Plugin](https://rook.io/docs/rook/latest/Troubleshooting/krew-plugin)

Reviewer 2: ```suggestion
  For more details, see the [Rook Krew Plugin](https://rook.io/docs/rook/latest/Troubleshooting/krew-plugin)
```

Reviewer 3: "latest" in the url will Point to the latest master build docs and not the latest release version. To point to the latest release version, use "latest-release".

Author: @galexrt check the link I have updated
